frontend/src/ui/components/ChatBubble/ChatBubble.scss

```diff
@@ -0,0 +1,45 @@
+.chat-bubble {
+    .text-cont {
+        position: absolute;
+        .text {
+            font-size: small;
+            color: black;
```

Comment:
add color to colors.scss

Reply:
i don’t think adding a simple black color to variables is necessary

Reply:
Using variables, even for built-in colors, is a good practice. It improves code clarity and maintainability by enabling global color changes. This approach allows for easy updates to colors across the entire application.